### ISSUE TITLE
在graphgym文件下，为找到 train.py 和 eval.py

### ISSUE THREAD

**Issue by sunmingqi-1** [DrZiji/VecFloorSeg#5]
![image](https://github.com/DrZiji/VecFloorSeg/assets/79261103/36f961d0-40c7-49e0-8769-ff6e23723c76)

**Comment by save-me-ctrl-c** [comment]
朋友，你找到他的require文件在哪里吗，我这边找不到，不知道是不是作者更新把它删了，有的话，方便发一下吗，邮箱是lizhiheng444@gmail.com，万分感谢

**Comment by sunmingqi-1** [comment]
> 朋友，你找到他的require文件在哪里吗，我这边找不到，不知道是不是作者更新把它删了，有的话，方便发一下吗，邮箱是[lizhiheng444@gmail.com](mailto:lizhiheng444@gmail.com)，万分感谢

你首先要install pytorch_geometric, 之后Replace the graphgym and torch_geometric in pyg with corresponding dir in our repository  
![image](https://github.com/DrZiji/VecFloorSeg/assets/79261103/5cb15424-4259-48b8-ad8b-8c5e8a2ac2e8)

**Comment by BobbyZ04** [comment]
> > 朋友，你找到他的require文件在哪里吗，我这边找不到，不知道是不是作者更新把它删了，有的话，方便发一下吗，邮箱是[lizhiheng444@gmail.com](mailto:lizhiheng444@gmail.com)，万分感谢
> 
> 你首先要install pytorch_geometric, 之后Replace the graphgym and torch_geometric in pyg with corresponding dir in our repository ![image](https://private-user-images.githubusercontent.com/79261103/296247237-5cb15424-4259-48b8-ad8b-8c5e8a2ac2e8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4MDkzMTcsIm5iZiI6MTcxMDgwOTAxNywicGF0aCI6Ii83OTI2MTEwMy8yOTYyNDcyMzctNWNiMTU0MjQtNDI1OS00OGI4LWFkOGItOGM1ZThhMmFjMmU4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE5VDAwNDMzN1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYzYjQ2NDg2N2Q4NDA4MWZjMjMyNjllMWU2NjA5ZDUzNzFlODI4ZDkzY2EwM2Q2OWNlZDkwOGE1OTgwMDlkMTImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.j1fENhy0GOiOtPF-j8gHnOFMiajOR1kYUNJIhvA-XYI)

想问下你的requirement.txt在哪里哦，谢谢

**Comment by dawang-11** [comment]
想问下 data preparation具体怎么操作

**Comment by yangfy2023** [comment]
> 朋友，你找到他的require文件在哪里吗，我这边找不到，不知道是不是作者更新把它删了，有的话，方便发一下吗，邮箱是[lizhiheng444@gmail.com](mailto:lizhiheng444@gmail.com)，万分感谢

请问现在找到requirement.txt了吗

**Comment by w243943** [comment]
> > 朋友，你找到他的require文件在哪里吗，我这边找不到，不知道是不是作者更新把它删了，有的话，方便发一下吗，邮箱是[lizhiheng444@gmail.com](mailto:lizhiheng444@gmail.com)，万分感谢
> 
> 请问现在找到requirement.txt了吗

请问您知道怎么解决了吗

**Comment by w243943** [comment]
> Replace the graphgym and torch_geometric in pyg with corresponding dir in our repository
请问您知道怎么解决了吗

**Comment by tinxiaoao** [comment]
我的复现出现了一点问题，安装的源文件中graphgym and torch_geometric是需要删除掉，然后将项目中的文件夹复制过去吗？还是直接合并呢

**Comment by dawang-11** [comment]
> 我的复现出现了一点问题，安装的源文件中graphgym and torch_geometric是需要删除掉，然后将项目中的文件夹复制过去吗？还是直接合并呢

我当时也疑惑这个问题（感觉没说清楚，

**Comment by sunmingqi-1** [comment]
是的安装后复制过去


| |
孙明奇
|
|
***@***.***
|
---- 回复的原邮件 ----
| 发件人 | ***@***.***> |
| 发送日期 | 2024年7月23日 20:21 |
| 收件人 | ***@***.***> |
| 抄送人 | ***@***.***> ,
***@***.***> |
| 主题 | Re: [DrZiji/VecFloorSeg] 在graphgym文件下，为找到 train.py 和 eval.py (Issue #5) |

我的复现出现了一点问题，安装的源文件中graphgym and torch_geometric是需要删除掉，然后将项目中的文件夹复制过去吗？还是直接合并呢

我当时也疑惑这个问题（感觉没说清楚，

**Comment by w243943** [comment]
> 是的安装后复制过去 | | 孙明奇 | | ***@***.*** | ---- 回复的原邮件 ---- | 发件人 | ***@***.***> | | 发送日期 | 2024年7月23日 20:21 | | 收件人 | ***@***.***> | | 抄送人 | ***@***.***> , ***@***.***> | | 主题 | Re: [DrZiji/VecFloorSeg] 在graphgym文件下，为找到 train.py 和 eval.py (Issue #5) | 我的复现出现了一点问题，安装的源文件中graphgym and torch_geometric是需要删除掉，然后将项目中的文件夹复制过去吗？还是直接合并呢 我当时也疑惑这个问题（感觉没说清楚， — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.Message ID: ***@***.***>

复制过去之后也没有test和val，请问这个什么原因呢

**Comment by tinxiaoao** [comment]
python graphgym/train.py --cfg graphgym/configs/CUBI.yaml --seed 0
请问这一步是不是写错了，应该是：
python graphgym/main.py --cfg graphgym/configs/CUBI.yaml --seed 0呢？我最近都在调试这个代码，但是都遇到了问题，不知道有没有朋友实现了复现

**Comment by yyw13791768446** [comment]
> python graphgym/train.py --cfg graphgym/configs/CUBI.yaml --seed 0 请问这一步是不是写错了，应该是： python graphgym/main.py --cfg graphgym/configs/CUBI.yaml --seed 0呢？我最近都在调试这个代码，但是都遇到了问题，不知道有没有朋友实现了复现

请问您复现出来了这个源码了嘛？

**Comment by addiwang** [comment]
author rewrite torch_geometric 2.0.4, copy  directory "torch_geometric " in this project  to replace  torch_geometric in site-package, run python graphgym/train.py --cfg graphgym/configs/CUBI.yaml  in root directory

**Comment by Dazai0x** [comment]
请问requirement.txt在哪？

**Comment by Dazai0x** [comment]
> 我的复现出现了一点问题，安装的源文件中graphgym and torch_geometric是需要删除掉，然后将项目中的文件夹复制过去吗？还是直接合并呢

请问您现在复现成功了吗？

**Comment by w243943** [comment]
没有，开源也不回复问题，我服啦，换一个方法吧发自我的荣耀手机-------- 原始邮件 --------发件人： Dazai0x ***@***.***>日期： 2024年12月2日周一 12:00收件人： DrZiji/VecFloorSeg ***@***.***>抄送： Wang Yumeng ***@***.***>, Comment ***@***.***>主    题： Re: [DrZiji/VecFloorSeg] 在graphgym文件下，为找到 train.py 和 eval.py (Issue #5)

我的复现出现了一点问题，安装的源文件中graphgym and torch_geometric是需要删除掉，然后将项目中的文件夹复制过去吗？还是直接合并呢

请问您现在复现成功了吗？

**Comment by Dazai0x** [comment]
> 没有，开源也不回复问题，我服啦，换一个方法吧发自我的荣耀手机-------- 原始邮件 --------发件人： Dazai0x ***@***.***>日期： 2024年12月2日周一 12:00收件人： DrZiji/VecFloorSeg ***@***.***>抄送： Wang Yumeng ***@***.***>, Comment ***@***.***>主 题： Re: [DrZiji/VecFloorSeg] 在graphgym文件下，为找到 train.py 和 eval.py (Issue #5) 我的复现出现了一点问题，安装的源文件中graphgym and torch_geometric是需要删除掉，然后将项目中的文件夹复制过去吗？还是直接合并呢 请问您现在复现成功了吗？ —Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you commented.Message ID: ***@***.***>

能推荐一下其他的平面图分析方法吗？我现在复现成功了dfpr，但感觉他是21年的年限太老了

**Comment by awesomeNabi** [comment]
> python graphgym/train.py --cfg graphgym/configs/CUBI.yaml --seed 0 请问这一步是不是写错了，应该是： python graphgym/main.py --cfg graphgym/configs/CUBI.yaml --seed 0呢？我最近都在调试这个代码，但是都遇到了问题，不知道有没有朋友实现了复现

同问啊，请问您复现出来了吗
